src/Traits/MutatesSeedCommands.php

```diff
@@ -51,7 +51,7 @@ public function handle()
         $this->input->setOption('force', true);
         $this->input->setOption('database', $this->connection->tenantName());
 
-        if (! $this->option('class')) {
+        if (config('tenancy.db.tenant-seed-class')) {
```

Comment:
This doesn't allow users to change the seeder class via the command.  You still need to check for the `class` option somehow.

Reply:
I agree with Eric, the root cause to fix is identifying why `$this->option('class')` is not a boolean false or null when you leave out `--class` while calling the command.